Feat:Spacebattle gateway rework

Ykipss opened the pull request:
## Что ПР делает
Перерабатывает старый гейт spacebattle.dmm или же Nanotrasen Cruiser.

Почему это хорошо для игры
Вообще по-сути я это уже перечислял в предложке, прошедшей авоут. Но могу всё-же объяснить причину почему я взялся за переработку гейта:Слишком мало по-настоящему сложных гейтов. Из сложных врат я могу вспомнить только шахты дикого запада, что достаточно плохо.
Если рассудить здраво, гейты - Подземелья, которые исследователю( или попавшему туда бедолаге ) нужно пройти что-бы налутаться вещей или попросту выжить.
Возьму как пример не сложного гейта:academy.dmm AKA Академия магов. В ней все вещи исследователю или проходимцу даются чуть-ли не на блюде, а чуть-ли не единственные абсолютно не опасные враги это плачущие статуи из доктора кто( мимиков можно даже не встретить ).

## Сам гейт
![2022 07 29-00 50 09](https://user-images.githubusercontent.com/109545437/182091469-c25f21f0-c03c-4540-aa47-5a127da66e23.png)


## Changelog
🆑
Увеличено ХП мобов.
Добавлено больше стартового оружия.
Проведена проводка для возможности запитать корабль.
Все отсеки корабля( не включая мед, кухню и мостик ) были изменены.
/🆑

## Post Scriptum
Во-первых, лут с мобов никак не изменён. ( С синдикатовцев падают мечи и щиты )
Во-вторых, вот предложка прошедшая авоут:https://discord.com/channels/617003227182792704/755125334097133628/1002235056984698931

Ykipss committed:
Готово(?)

Я просто перенёс всё то, что сделал ранее

Bizzonium commented on code/modules/awaymissions/corpse.dm:
```diff
@@ -448,6 +448,12 @@
 	id_job = "Scientist"
 	outfit = /datum/outfit/job/scientist
 
+/obj/effect/mob_spawn/human/securty
```
```suggestion
/obj/effect/mob_spawn/human/security
```

Bizzonium commented on code/modules/awaymissions/corpse.dm:
```diff
@@ -448,6 +448,12 @@
+/obj/effect/mob_spawn/human/securty
+	name = "Security Officer"
+	mob_name = "Security Officer"
+	id_job = "Security Officer"
```
`id_job` точно правильный для профессии?

Ykipss replied:
Сорян, опечатался ._."

Ykipss committed:
Изменение

Добавил где нужно АПЦ ( в меде и на выходе верхнем в космос )
А так-же поменял ХП мобов оставшихся, где ХП было не изменено

Ykipss committed:
MaxHealth

Изменил максимальное здоровье у всех синдикатовцев на корабле, исключая синдикатовцев вне нанотрейзенского корабля
